mypy/build.py

```diff
@@ -2286,9 +2247,11 @@ def verify_dependencies(self, suppressed_only: bool = False) -> None:
             all_deps = self.suppressed
         else:
             # Strip out indirect dependencies. See comment in build.load_graph().
-            dependencies = [dep for dep in self.dependencies
-                            if self.priorities.get(dep) != PRI_INDIRECT]
             all_deps = dependencies + self.suppressed + self.ancestors
+            all_deps = [dep for dep in all_deps
+                        if self.priorities.get(dep) != PRI_INDIRECT]
+            #all_deps = dependencies + self.suppressed + self.ancestors
```

Comment:
Can we remove this commented line? Or alternatively include a comment explaining why we're keeping it/linking to this PR/describing the experiment-in-process.

Reply:
Oh, yeah, oops